src/containers/articles/ShareArticle.js

```diff
@@ -0,0 +1,125 @@
+import React, { Component } from "react";
+import { apiCalls } from "../../Helpers/axios";
+import Input from "../../components/Input/Input";
+import Modal from "../../components/Modal/Modal";
+export class ShareBar extends Component {
+  state = {
+    openmodal: false,
+    email: ""
+  };
+  handleSuccessShareResponse = ({ data }) => {
+    const newLink = data.shared_link.replace(
+      "http://ah-premier-staging.herokuapp.com/api",
+      "https://ah-fortem-staging.herokuapp.com"
+    );
+    const newWindow = window.open(newLink, "_blank");
+    newWindow.focus();
+  };
+
+  handleEmailChange = event => {
+    this.setState({
+      email: event.target.value
+    });
+  };
+
+  handleShareSocial = (slug, platform, data = {}) => () => {
+    const token = localStorage.getItem("token");
+    return apiCalls("post", "/articles/" + slug + "/share/" + platform, data, {
+      Authorization: `Bearer ` + token
+    }).then(this.handleSuccessShareResponse);
+  };
+  handleShareEmail = slug => {
+    const token = localStorage.getItem("token");
+    return apiCalls(
```

Comment:
This could also use a success and failure response displayed to the user.

Reply:
.then and .catch successfully added with messages added to Snackbar